README.md

```diff
@@ -47,7 +47,6 @@ $ git clone https://github.com/thingsapi/things.py && cd things.py && make insta
 [{'uuid': 'hIo1FJlAYGKt1Yj38vzKc3',
   'type': 'area',
   'title': 'Family',
-  'status': 'open',
```

Comment:
no status information in the model?

Reply:
It's an area type. Areas don't have a status afaik.

Reply:
Right...

Reply:
Then imho the other example ("'Make reservation for dinner'") should have a status

Reply:
If it's a task or project, then status is included. I might have dropped it from the example since it was getting really long. :)